src/main/java/app/bpartners/geojobs/endpoint/event/gen/DetectionJobSaved.java

```diff
@@ -0,0 +1,24 @@
+package app.bpartners.geojobs.endpoint.event.gen;
+
+import app.bpartners.geojobs.repository.model.geo.tiling.TilingTask;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.annotation.processing.Generated;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Generated("EventBridge")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Data
+@EqualsAndHashCode
+@ToString
+public class DetectionJobSaved implements Serializable {
```

Comment:
Bad. @Mahefaa or @NyH1409 > why is this bad?

Reply:
@Data is not really needed + the naming induces that it's a trigger after a DetectionJob is saved, not explicit enough